### PR TITLE
make a pointer pubSub field of Source

### PR DESCRIPTION
**Pull request by voscob**
### Description

Please include a summary of the change and what type of change it is (new feature, bug fix, refactoring, documentation).
Please also include relevant motivation and context.
List any dependencies that are required for this change.

Fixes # (issue)

### Quick checks:

- [ ] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [ ] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-gcp-pubsub/pulls) for the same update/change.
- [ ] I have written unit tests.
- [ ] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: clients/pubsub.go [hunk 1]
```diff
@@ -33,20 +33,20 @@ type PubSub struct {
 }
 
 // NewClient initializes a Pub/Sub client and starts receiving a messages to struct channels.
-func NewClient(ctx context.Context, cfg config.Source) (PubSub, error) {
+func NewClient(ctx context.Context, cfg config.Source) (*PubSub, error) {
 	const maxOutstandingMessages = 1000
```
Comment (maksenius) [inline clients/pubsub.go:37]: maybe lets move it from function

Comment (voscob) [reply clients/pubsub.go:37]: Did you mean to move it to the `Open` method?

Comment (maksenius) [reply clients/pubsub.go:37]: sorry I mean

const maxOutstandingMessages = 1000

Comment (maksenius) [reply clients/pubsub.go:37]: ok
lets save it here

**Review by maksenius — APPROVED** [review]

**Pull request merged**